openstack/identity/v3/projects/requests.go

```diff
@@ -243,3 +243,30 @@ func Update(ctx context.Context, client *gophercloud.ServiceClient, id string, o
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// GetProjectTag checks if a a tag exists in a project.
+func GetProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r GetProjectTagResult) {
+	resp, err := client.Get(ctx, getProjectTagURL(client, projectID, tag), nil, &gophercloud.RequestOpts{
+		OkCodes: []int{204},
+	})
+	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
+	return
+}
+
+// AddProjectTag adds a tag in a project.
+func AddProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r AddProjectTagResult) {
```

Comment:
This should take a list of tags. Also, drop `Project` from the function name and make this plural. This should then be called `AddTags()`.

Reply:
Regarding this, we are implementing `Add single tag to a project` which only takes one tag , and not  `Modify tag list for a project` which takes a list of tags.  Therefore, I think the name `AddTag()` is more suitable and by extension I dont think we need to change the tag argument to a list.

Reply:
Yeah, you're right. I've somehow missed the API call that takes a tag. That being said, it shouldn't be difficult to add the variants that take a list of tags and perhaps it would be worth it to add it there too. Same for the variant of Delete that drops all tags. What do you think?

Reply:
I agree that it wouldn't be too much of a hustle to add the list and delete all tags API calls. But, I would rather adding the rest of the project tag of the API suite in another PR, because we need the three API calls of this MR kinda quickly.